src/pages/landing/stocks/_dtrading.js

```diff
@@ -0,0 +1,169 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { graphql, useStaticQuery } from 'gatsby'
+import device from 'themes/device'
+import { Container, SectionContainer } from 'components/containers'
+import { Header, Text, QueryImage } from 'components/elements'
+
+const StyledSection = styled(SectionContainer)`
+    background-color: var(--color-white);
+    @media ${device.tabletL} {
+        padding: 1.74rem 0 4rem 0;
+    }
+`
+const StyledContainer = styled(Container)`
+    display: flex;
+    flex-direction: column;
```

Comment:
The `Container` component has a default display of `flex`. You can also pass in the flex-direction as `fd` props to `Container`. You can omit this StyledContainer completely.

Reply:
StyledContainer ommited to Container.